src/core/hoc/StylesInjection.tsx

```diff
@@ -3,15 +3,17 @@ import { StyleSheetManager } from 'styled-components';
 import createCache from '@emotion/cache';
 import { CacheProvider } from '@emotion/react';
 
-interface Props extends React.HTMLProps<HTMLDivElement> {
-    id: string;
+interface Props extends Pick<React.HTMLProps<HTMLDivElement>, 'children'> {
     injectionPoint?: HTMLElement | undefined;
 }
 
 export const StylesInjection = (props: Props) => {
 
+  // @emotion/cache only accept alpha characters
+  const uniqueAlphaCharacterId = Math.random().toString(36).replace(/[^a-z]+/g, '').substring(0, 10);
```

Comment:
Maybe you could use a lib to generate the ID? In Nexus we use https://github.com/dylang/shortid

Reply:
Ok, says it's deprecated so let's use `nanoid`.

However, @emotion/cache only accepts alpha characters so still have to do some conversion.

Reply:
Thanks for replacing shortId by nanoid :)